.github/workflows/long-running-azure.yaml

```diff
@@ -515,6 +515,7 @@ jobs:
             --yes --verbose
       - name: Clean up cluster
         if: always()
+        timeout-minutes: 60
```

Comment:
Added timeout to clean up step.

Reply:
What's the goal of the timeout? That we'd be notified if we hit it?

Reply:
Default is 6 hours, so fail faster?

Reply:
Exactly. To fail faster.